src/file_pipe_log/mod.rs

```diff
@@ -52,6 +53,17 @@ pub mod debug {
         items: VecDeque<LogItem>,
     }
 
+    impl<B: FileBuilder> Iterator for LogItemReader<B> {
+        type Item = LogItem;
```

Comment:
Should be `Result<LogItem>`, or the dump will stop once encountering the first corruption.

Reply:
@tabokie  We should dump all the log and omit error in the process? and should we dump the error info?

Reply:
We should print the error and continue.